Backend/templates/gitlab-ci/master-gitlab-ci.yml

```diff
@@ -249,12 +289,12 @@ DeployDocumentationProd:
 
     - helm init --client-only
 
-    #TODO check if this is correct to have CHART_NAME here
-    - helm upgrade $CHART_NAME ./$CHART_NAME 
+    - helm upgrade $NAME ./chart/$NAME
       --install
       --version $BUILD_NUMBER
       --set image.repository=$CI_REGISTRY/$CI_PROJECT_PATH/$REGISTRY_PATH
       --set image.tag=$BUILD_NUMBER
+      --set "image.env[0].name=ENV,image.env[0].value=$ENVIRONMENT"
```

Comment:
Should the `ENV` variable (that is passed to the docker) be with another name? like `ENVIRONMENT`

Reply:
i think `ENV` is ok 🚀